docs/content/preview/explore/transactions/isolation-levels.md

```diff
@@ -32,38 +32,58 @@ type: docs
+## Isolation levels in PostgreSQL and YSQL
+
+PostgreSQL (and the SQL standard) has four isolation levels: Serializable, Repeatable read, Read committed, and Read uncommitted.
+
+The following table shows the mapping between the PostgreSQL isolation levels in YSQL, along with transaction anomalies that can occur at each isolation level:
 
-* Dirty read: A transaction reads data written by a concurrent uncommitted transaction.
+| PostgreSQL Isolation | YugabyteDB Equivalent     | Dirty Read | Non-repeatable Read | Phantom Read | Serialization Anomaly |
+| :------------------- | :------------------------ | :--------- | :------------------ | :----------- | :-------------------- |
+| Read uncommitted | Read Committed {{<badge/tp>}} | Allowed, but not in YSQL |  Possible | Possible | Possible |
+| Read committed   | Read Committed {{<badge/tp>}} | Not possible | Possible     | Possible | Possible |
+| Repeatable read  | Snapshot                      | Not possible | Not possible | Allowed, but not in YSQL | Possible |
+| Serializable     | Serializable                  | Not possible | Not possible | Not possible | Not possible |
 
-* Nonrepeatable read: A transaction rereads data it has previously read and finds that data has been modified by another transaction committed after the initial read.
+The strictest isolation level is Serializable, which requires that any concurrent execution of a set of Serializable transactions is guaranteed to produce the same effect as running them in some serial order (one transaction at a time).
 
-* Phantom read: A transaction re-executes a query returning a set of rows that satisfy a search condition and finds that the set of rows satisfying the condition has changed due to another recently-committed transaction.
+The other levels are defined by which anomalies (none of which are possible with serializable) must not occur as a result of interactions between concurrent transactions. Due to the definition of Serializable isolation, none of these anomalies are possible at that level.
 
-* Serialization anomaly: The result of successfully committing a group of transactions is inconsistent with all possible orderings of running those transactions one at a time.
+The following are the various transaction anomalies:
+
+- Dirty read: A transaction reads data written by a concurrent uncommitted transaction.
+
+- Non-repeatable read: A transaction rereads data it has previously read and finds that data has been modified by another transaction committed after the initial read.
```

Comment:
it has previously read -> it has previously fetched
(because what was read but discarded by filters may change and this is accepted in repeatable reads)